test/java/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelTest.java

```diff
@@ -68,20 +70,32 @@ public void choose_pem_file_button_is_swapped_for_a_clean_button_that_cleans_the
   }
 
   @Test
-  public void shows_error_and_clears_field_after_inserting_an_invalid_start_date() {
-    component.setEndDate(LocalDate.of(2018, 1, 1));
-    component.setStartDate(LocalDate.of(2020, 1, 1));
-    assertThat(component.errorDialog(500), is(GenericUIMatchers.visible()));
-    component.acceptErrorDialog();
-    assertThat(component.startDateField(), is(empty()));
+  public void cannot_insert_an_invalid_end_date() {
```

Comment:
I'd suggest renaming this to explicitly tell the condition which makes impossible to insert an invalid date: `cannot_insert_an_end_date_after_the_established_start_date` or something like that

Reply:
Renamed to `cannot_insert_an_end_date_before_the_set_start_date`